packages/bpk-component-modal/src/BpkModal.js

```diff
@@ -52,6 +52,7 @@ const BpkModal = props => {
       onClose={onClose}
       target={target}
       renderTarget={renderTarget}
+      closeOnEscPressed={closeOnScrimClick}
```

Comment:
`closeOnScrimClick` is not he same as `closeOnEscPressed ` - they should be treated as separate props!

Reply:
Wouldn't we want them to share the same logic in this case?

Reply:
We could enable it by default in the Dialog, but the configuration option in the modal should be separate from `closeOnScrimClick`.

It's a surprising side effect for consumers who disable `closeOnScrimClick` and also inadvertently disable that ability to close on esc

Reply:
Cool, I have set it to share the logic with `dismissible` in the dialog component.